sopel/module.py

```diff
@@ -8,4 +8,38 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from sopel.plugin import *  # noqa
+# Import everything from sopel.plugin at the time of replacement.
+# Everything new from this point on must *not* leak here.
+# Therefore, don't add anything to this import list. Ever.
+from sopel.plugin import (  # noqa
+    ADMIN,
+    action_commands, commands,
```

Comment:
```suggestion
    action_commands,
    commands,
```

I'd expect flake8 to flag this. Interesting.

Reply:
It can't, I used `# noqa`.

Reply:
Uh duh 🤦‍♂️

Carry on!